src/macros.jl

```diff
@@ -1508,7 +1511,40 @@ macro NLexpression(args...)
     end)
 end
 
-# syntax is @NLparameter(m, p[i=1] == 2i)
+"""
+    @NLparameter(model, param == value)
+
+Creates and returns a nonlinear parameter `param` attached to the model `model`
```

Comment:
We usually use `Return` in docstrings so for consistency, it should be `Create and return`

Reply:
Done